src/JestProcessManagement/JestProcess.ts

```diff
@@ -75,7 +75,7 @@ export class JestProcess {
 
     const options = {
       noColor: true,
-      reporters: ['default', reporterPath],
+      reporters: ['default', `'${reporterPath}'`],
```

Comment:
Or should we do a `${reporterPath.replace("'","\\'")}`? If not, I'd be also fine with the current solution.